### PR TITLE
Add Functionality to Include docker_params

### PR DESCRIPTION
**Pull request by gisjedi**
allow for docker_params to be added

### CODE REVIEW

**Comment by gisjedi** [comment]
Accomplished rebasing #582 on top of my master branch updates to correct Travis pull requests.

File: scale/mesos_api/tasks.py [hunk 1]
```diff
@@ -88,7 +88,10 @@ def _create_docker_task(task):
     mesos_task.container.type = mesos_pb2.ContainerInfo.DOCKER
     mesos_task.container.docker.image = task.docker_image
     for param in task.docker_params:
-        mesos_task.container.docker.parameters.add(key=param.flag, value=param.value)
+        a_docker_param = mesos_task.container.docker.parameters.add()
+        a_docker_param.key = param.flag
+        a_docker_param.value = param.value
```
Comment (dfaller) [inline scale/mesos_api/tasks.py]: What is the reason for this change? The one line option is more succinct.

File: scale/job/models.py [hunk 2]
```diff
@@ -1052,6 +1052,10 @@ def schedule_job_executions(self, framework_id, job_executions, workspaces):
             job_exe.started = started
             job_exe.node = node
             job_exe.configure_docker_params(framework_id, workspaces)
+            
+            for key, value in job_exe.job.job_type.docker_params:
+                job_exe.configuration['job_task']['docker_params'].append({'flag': key, 'value': value})
+
```
Comment (dfaller) [inline scale/job/models.py]: 1. Please move this to JobExecution.configure_docker_params(). This is the location where Docker parameters are set up for the job execution.
2. Please use configuration.add_job_task_docker_param(DockerParam(key, value)) instead of directly hitting the internals of the configuration dict.
3. Please add a conditional to ensure that docker_params is not None. It's possible for the field to be set to None.

Comment (antman1p) [inline scale/job/models.py]: For clarrification.  Does this mean that you want us to do the changes you are asking above, or dod you already make the changes?

Comment (dfaller) [inline scale/job/models.py]: We are asking you to make the changes. Thanks!

**Comment by dfaller** [comment]
Due to conflicts with master, this functionality has been merged into master via a different PR.
